src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/FileApiInteropRestore.depproj

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
what is getting it into the bindir? ie., where does it come from/get checked in to?